content/getting-started/finding-kata.md

```diff
@@ -6,8 +6,44 @@ next: "/getting-started/solving-kata/"
 
 # Finding Kata
 
-<!---
-TODO training suggestions
-TODO kata search page
-TODO kata "numbers": rank, satisfaction, possibly others (which?) as criteria for selecting a kata to solve
--->
+Your account is configured and you are ready for your next challenge. Now you will learn how to find a task to train on.
+
+## Codewars trainer
+
+The easiest way of getting started is to use our trainer on the dashboard to pick your next kata.
+
+![training routines](./img/finding-kata_01_training-routines.png)
+
+The trainer allows you to pick your language, select your focus, preview the challenge, skip to another challenge and to start training.
```

Comment:
```suggestion
The suggestion area allows you to preview the suggested challenge, pick your language, select your focus, then to start training or skip to another challenge.
```
to keep the order with the picture (just the preview is a bit weird to place... let's try it in first position)